### PR TITLE
Quick access

### PR DESCRIPTION
**Pull request by chan227**

### CODE REVIEW

**Comment by chan227** [comment]
ready

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -26,17 +26,17 @@
         <div class="card-header tab-card-header">
           <ul class="nav nav-tabs card-header-tabs" id="myTab" role="tablist">
             <li class="nav-item">
-                <a class="nav-link" id="one-tab" data-toggle="tab" href="#one" role="tab" aria-controls="One" aria-selected="true">Post</a>
+              <a class="nav-link" id="one-tab" data-toggle="tab" href="#one" role="tab" aria-controls="One" aria-selected="true">Post</a>
```
Comment (saobv-2223) [inline app/views/users/show.html.erb:29]: You should use link_to instead of a tag if you can

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -71,23 +71,22 @@
...
                 </div>
               </div>
             <% end %>
           </div>
 
           <div class="tab-pane fade p-4" id="four" role="tabpanel" aria-labelledby="four-tab">
-            <h3><%= t ".report_count" %> (<%= @user.activities.report.count %>)</h3>
-            <% @user.activities.report.each do |activity| %>
+            <h3><%= t ".report_count" %> (<%= Activity.report.count %>)</h3>
+            <% Activity.report.each do |activity| %>
```
Comment (saobv-2223) [inline app/views/users/show.html.erb:82]:
```suggestion
            <% activity.report.each do |activity| %>
```

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -71,23 +71,22 @@
...
+                  <%= link_to activity.post.title, post_path(activity.post) %>
                 </div>
               </div>
             <% end %>
           </div>
 
           <div class="tab-pane fade p-4" id="four" role="tabpanel" aria-labelledby="four-tab">
-            <h3><%= t ".report_count" %> (<%= @user.activities.report.count %>)</h3>
-            <% @user.activities.report.each do |activity| %>
+            <h3><%= t ".report_count" %> (<%= Activity.report.count %>)</h3>
```
Comment (saobv-2223) [inline app/views/users/show.html.erb:81]:
```suggestion
            <h3><%= t ".report_count" %> (<%= activity.report.count %>)</h3>
```

File: app/views/users/show.html.erb (hunk 2 above)
Comment (chan227) [reply app/views/users/show.html.erb:82]: error occured

File: app/views/users/show.html.erb (hunk 3 above)
Comment (chan227) [reply app/views/users/show.html.erb:81]: error occurred.

Comment (chan227) [reply app/views/users/show.html.erb:81]: image

File: app/views/users/show.html.erb (hunk 1 above)
Comment (chan227) [reply app/views/users/show.html.erb:29]: `<%= link_to "#one", "id": "one-tab", {class: "nav-link", data: { "toggle" => "tab"}, "role": "tab", "aria-controls": "One", "aria-selected": true} do %> Post`

Comment (chan227) [reply app/views/users/show.html.erb:29]: I am not sure how to do it, can you teach me to do one?

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -30,30 +30,39 @@
...
-                <li><%= link_to t(".setting_header"), edit_user_path(current_user) %></li>
-                <li class="divider"></li>
-                <li>
-                <%= link_to t(".logout_header"), logout_path, method: :delete %>
-                </li>
+              <li><%= link_to t(".profile_header"), current_user %></li>
+              <li><%= link_to t(".setting_header"), edit_user_path(current_user) %></li>
+              <li class="divider"></li>
+              <li>
+              <%= link_to t(".logout_header"), logout_path, method: :delete %>
```
Comment (saobv-2223) [inline app/views/layouts/_header.html.erb]:
```suggestion
                <%= link_to t(".logout_header"), logout_path, method: :delete %>
```

File: app/views/posts/index.html.erb [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
-      <h2><%= post.body %></h2>
-    </p>
-    <%= t "posts.created_at" %>
-    <%= link_to post.user.name, post.user %>
-    <%= time_ago_in_words post.created_at %>
-    <%= t "posts.ago" %><br>
-    <%= link_to t("read.more"), post_path(post), class: "btn btn-default" %>
+
+<div class="col-md-6">
+  <% provide :title, t("posts.title") %>
```
Comment (saobv-2223) [inline app/views/posts/index.html.erb]: You should write provide at the top of view

File: app/views/posts/index.html.erb [hunk 6]
```diff
@@ -1,22 +1,25 @@
...
+      </p>
+      <h5>
+        <%= t "posts.created_at" %>
+        <%= link_to post.user.name, post.user %>
+        <%= time_ago_in_words post.created_at %>
+        <%= t "posts.ago" %><br>
+      </h5>
+      <%= link_to t("read.more"), post_path(post), class: "btn btn-default" %>
+    </div>
+  <% end %>
```
Comment (saobv-2223) [inline app/views/posts/index.html.erb]: you should write partial if you can

File: app/views/layouts/_header.html.erb (hunk 4 above)
Comment (chan227) [reply app/views/layouts/_header.html.erb]: done

File: app/views/posts/index.html.erb (hunk 5 above)
Comment (chan227) [reply app/views/posts/index.html.erb]: done

File: app/views/posts/index.html.erb (hunk 6 above)
Comment (chan227) [reply app/views/posts/index.html.erb]: done

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Pull request merged**